tool/main.cpp

```diff
@@ -394,7 +394,7 @@ int main(int argc, char* argv[])
          po::value<std::string>()->default_value(""),
          "a prefix that will be added to all output files")
         ("output.format",
-         po::value<std::vector<std::string>>()->default_value(std::vector<std::string>{"html"}, "{html}"),
+         po::value<std::vector<std::string>>()->default_value(std::vector<std::string>{"commonmark"}, "{commonmark}"),
```

Comment:
This probably should be in a separate commit.

Reply:
In principle I agree but I already add a Dockerfile and fix the README in the same commit. Sorry for that little mess.